AffirmSDK/AffirmCheckoutViewController.m

```diff
@@ -189,10 +189,12 @@ - (void)webView:(WKWebView *)webView decidePolicyForNavigationAction:(WKNavigati
     NSURL *URL = navigationAction.request.URL;
     NSString *strippedURL = [NSString stringWithFormat:@"%@://%@%@", URL.scheme, URL.host, URL.path];
     if ([strippedURL isEqualToString:AFFIRM_CHECKOUT_CONFIRMATION_URL]) {
+        BOOL isResponded = NO;
```

Comment:
Can we have a better name for this, maybe `hasRespondedWithCheckoutToken`?

Reply:
Okay.